libnetwork/internal/kvstore/boltdb/boltdb.go

```diff
@@ -34,9 +35,20 @@ func New(path, bucket string) (store.Store, error) {
 	}
 
 	db, err := bolt.Open(path, filePerm, &bolt.Options{
-		Timeout: time.Minute,
+		// The bbolt package opens the underlying db file and then issues an
+		// exclusive flock to ensures that it can safely write to the db. If
+		// it fails, it'll re-issue flocks every few ms until Timeout is
+		// reached.
+		// This nanosecond timeout bypasses that retry loop and make sure the
+		// bbolt package returns an ErrTimeout straight away. That way, the
+		// daemon, and unit tests, will fail fast and loudly instead of
+		// silently introducing delays.
+		Timeout: time.Nanosecond,
 	})
 	if err != nil {
+		if errors.Is(err, bolt.ErrTimeout) {
+			return nil, errors.New("the underlying boltdb file is already held open")
```

Comment:
Should we include the path as part of the error message? (does the original `err` contain useful information, or is that just "timeout" without more context?)